components/gitpod-cli/BUILD.yaml

```diff
@@ -9,7 +9,21 @@ packages:
       - CGO_ENABLED=0
       - GOOS=linux
     deps:
+      - :version
       - components/supervisor-api/go:lib
       - components/gitpod-protocol/go:lib
+    argdeps:
+      - version
     config:
       packaging: app
+    prep:
+      - ["cp", "_deps/components-gitpod-cli--version/version.txt", "cmd/version.txt"]
```

Comment:
Should not we have just only one file and only declaration in go code that when is referenced in different places?

Otherwise it does not help, calling `gp version` does not use the same code as settings version on request headers.